Libraries/Text/Text/RCTTextView.m

```diff
@@ -159,12 +159,40 @@ - (void)drawRect:(CGRect)rect
   NSTextContainer *textContainer = layoutManager.textContainers.firstObject;
 
   NSRange glyphRange = [layoutManager glyphRangeForTextContainer:textContainer];
-  [layoutManager drawBackgroundForGlyphRange:glyphRange atPoint:_contentFrame.origin];
-  [layoutManager drawGlyphsForGlyphRange:glyphRange atPoint:_contentFrame.origin];
-
-  __block UIBezierPath *highlightPath = nil;
   NSRange characterRange = [layoutManager characterRangeForGlyphRange:glyphRange
                                                      actualGlyphRange:NULL];
+  // [TODO(OSS Candidate ISS#2710739)
```

Comment:
I think that the feature is awesome.

It feels that this property should be part of like `ParagraphAttributes` (see Fabric's `ParagraphAttributes`), not `TextAttributes`.

So, it should be applied to the whole text node. Or, do we have use cases where it should be applied only for some fragments?

Reply:
Well, the biggest reason we can't do that is because mac doesn't work with Fabric yet so we have no ParagraphAttributes 😢.  Otherwise, just to be parallel with the w3c `font-smooth` style and webkit `--webkit-font-smoothing` styles which can be applied at any level of granularity.